package.json

```diff
@@ -7,7 +7,7 @@
         "@sa11y/jest": "^0.3.1",
         "@salesforce/eslint-config-lwc": "^2.0.0",
         "@salesforce/eslint-plugin-lightning": "^0.1.1",
-        "@salesforce/sfdx-lwc-jest": "^0.10.4",
+        "@salesforce/sfdx-lwc-jest": "^0.12.6",
```

Comment:
@sella24 I think if you use the latest version ("^0.14.0"), you won't need the resolutions for `node-notifier` and `postcss`. Everything else looks good tho